js_modules/dagster-ui/packages/ui-core/src/workspace/RepositoryLocationsList.tsx

```diff
@@ -70,35 +70,35 @@ export const RepositoryLocationsList = ({loading, codeLocations, searchValue}: P
   return (
     <>
       <VirtualizedCodeLocationHeader />
-      <div style={{overflow: 'hidden'}}>
```

Comment:
It's sorta interesting that this fixes the problem, I sort of think of `overflow:hidden` == no scroll bars. Maybe removing the container div entirely had some other impact too? (Either way, I'm glad you got to the bottom of this, what a whacky bug)